Docs/sphinx/Spray.rst

```diff
@@ -226,6 +226,27 @@ The procedure is as follows for updating the spray droplet:
 
     :math:`N_{d}` is the number of droplets per computational parcel, and :math:`V_{\rm{cell}}` is the volume for the cell of interest. Note that the cell volume can vary depending on AMR level and if an EB is present.
 
+
+.. warning::
+
+   **Dimensional Assumptions in the Spray Model**
+
+   The spray model assumes spherically symmetric droplets and is inherently three-dimensional. 
+   For two-dimensional simulations, the domain is treated as one cell wide in the :math:`z`-direction, 
+   with :math:`L_z = \Delta x`, so the volume gas-phase source terms is:
+
+   .. math::
+
+      V_{\rm{cell}} = \Delta x \, \Delta y \, \Delta x
+
+   This effectively places an infinite array of droplets spaced :math:`\Delta x` apart in :math:`z`, 
```

Comment:
Add "For nominally single droplet cases,". Multidroplet cases in 2D would be even more confusing!